sdk/keyvault/azcertificates/client.go

```diff
@@ -0,0 +1,1568 @@
+}
+
+// ListCertificatesPage contains the current page of results for the Client.ListSecrets operation
+type ListCertificatesPage struct {
+	// READ-ONLY; A response message containing a list of certificates in the key vault along with a link to the next page of certificates.
+	Certificates []*CertificateItem `json:"value,omitempty" azure:"ro"`
+
+	// RawResponse contains the underlying HTTP response.
+	RawResponse *http.Response
+}
+
+// convert internal Response to ListCertificatesPage
+func listKeysPageFromGenerated(i generated.KeyVaultClientGetCertificatesResponse) ListCertificatesPage {
+	var vals []*CertificateItem
+
+	for _, v := range i.Value {
+		vals = append(vals, &CertificateItem{
+			Attributes:     certificateAttributesFromGenerated(v.Attributes),
+			ID:             v.ID,
+			Tags:           convertGeneratedMap(v.Tags),
+			X509Thumbprint: v.X509Thumbprint,
+		})
+	}
+
+	return ListCertificatesPage{
+		RawResponse:  i.RawResponse,
+		Certificates: vals,
+	}
+}
+
+// ListCertificates retrieves a list of the certificates in the Key Vault as JSON Web Key structures that contain the
+// public part of a stored certificate. The LIST operation is applicable to all certificate types, however only the
+// base certificate identifier, attributes, and tags are provided in the response. Individual versions of a
+// certificate are not listed in the response. This operation requires the certificates/list permission.
+func (c *Client) ListCertificates(options *ListCertificatesOptions) ListCertificatesPager {
+	return ListCertificatesPager{
+		genPager: c.genClient.GetCertificates(c.vaultURL, options.toGenerated()),
+	}
+}
+
+// ListCertificateVersionsPager is the pager returned by Client.ListCertificateVersions
+type ListCertificateVersionsPager struct {
+	genPager *generated.KeyVaultClientGetCertificateVersionsPager
+}
+
+// PageResponse returns the results from the page most recently fetched from the service.
+func (l *ListCertificateVersionsPager) PageResponse() ListCertificateVersionsPage {
+	return listKeyVersionsPageFromGenerated(l.genPager.PageResponse())
+}
+
+// Err returns an error value if the most recent call to NextPage was not successful, else nil.
+func (l *ListCertificateVersionsPager) Err() error {
+	return l.genPager.Err()
+}
+
+// NextPage fetches the next available page of results from the service. If the fetched page
+// contains results, the return value is true, else false. Results fetched from the service
+// can be evaluated by calling PageResponse on this Pager.
+func (l *ListCertificateVersionsPager) NextPage(ctx context.Context) bool {
+	return l.genPager.NextPage(ctx)
+}
+
+// ListCertificateVersionsOptions contains the options for the ListCertificateVersions operations
+type ListCertificateVersionsOptions struct {
+	// Maximum number of results to return in a page. If not specified the service will return up to 25 results.
+	MaxResults *int32
```

Comment:
Same here and anywhere `MaxResults` is exposed. Shouldn't even be necessary to declare since we don't pass it to the server (leaving the default in place).